scipy/stats/_morestats.py

```diff
@@ -1129,12 +1129,21 @@ def boxcox(x, lmbda=None, alpha=None, optimizer=None):
 
 def _boxcox_inv_lmbda(x, y):
     # compute lmbda given x and y for Box-Cox transformation
-    num = special.lambertw(-(x ** (-1 / y)) * np.log(x) / y, k=1)
+    num = special.lambertw(-(x ** (-1 / y)) * np.log(x) / y, k=-1)
```

Comment:
I'm going to put this in a separate PR. It looks like whether the `-1` or `0` branch should be chosen depends on `x`. This is a bug fix, so we need to get it in 112.